hs/app/reach/Main.hs

```diff
@@ -2450,6 +2451,149 @@ whoami = command "whoami" $ info f fullDesc
   where
     f = pure . script $ write [N.text| echo "$whoami'" |]
 
+type DeviceCode = Text
+
+-- https://stackoverflow.com/a/60790430
+newtype SomeValue = SomeValue String
+instance FromJSON SomeValue where
+  parseJSON = withObject "SomeValue" $ \o -> SomeValue <$> o .: "html_url"
+
+support :: Subcommand
+support = command "support" $ info (pure step1) d
+  where
+    d = progDesc "Upload index.rsh and index.mjs to help us troubleshoot!"
+
+    splitByAmpersands :: BSLC8.ByteString -> [Text]
+    splitByAmpersands s = T.splitOn "&" (pack $ BSLC8.unpack s)
+
+    splitByEqualsSigns :: Text -> [Text]
+    splitByEqualsSigns s = T.splitOn (pack "=") s
+
+    process :: BSLC8.ByteString -> [[Text]]
+    process gitHubResponseString = do
+      let stringArray = splitByAmpersands gitHubResponseString
+      map splitByEqualsSigns stringArray
```

Comment:
$ and drop the let and the do

Reply:
I believe [3f8de27](https://github.com/reach-sh/reach-lang/pull/762/commits/3f8de27080a80a9f0fa7976366d0c98efaa836b3) addresses this.